src/player.cpp

```diff
@@ -9059,12 +9314,10 @@ void player::do_read( item &book )
                 }
             }
 
-            if( skill_level == reading->level || !skill_level.can_train() ) {
-                if( learner->is_player() ) {
-                    add_msg( m_info, _( "You can no longer learn from %s." ), book.type_name().c_str() );
-                } else {
-                    cant_learn.insert( learner->disp_name() );
```

Comment:
This has been removed without replacement, doesn't that change the outcome of something?